bittensor/subtensor.py

```diff
@@ -5106,7 +5106,7 @@ async def get_subnet_burn_cost(self, block: Optional[int] = None) -> Optional[st
         if lock_cost is None:
             return None
 
-        return lock_cost
+        return int(lock_cost)
```

Comment:
The docs state this should be in Tao (which is usually float). Is it actually in Rao? Even if so, why are we changing the type here?